frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -256,6 +256,11 @@ function ReaderBookmark:onShowBookmark()
                 page = self.ui.pagemap:getXPointerPageLabel(page, true)
             else
                 page = self.ui.document:getPageFromXPointer(page)
+                local flow = self.ui.document:getPageFlow(page)
+                page = self.ui.document:getPageNumberInFlow(page)
+                if flow > 0 then
+                    page = T("[%1]%2", page, flow)
```

Comment:
Might need a translator's comment. Also screenshots would be nice. ;-)

Reply:
I confess I've used `T(...)` without knowing what it means... I don't think this should be translated though, so I guess I should use `("[%1]%2"):format(page, flow)`, as in the footer code?

Reply:
Actually you confused me by needlessly using the template function here when you're just doing string.format.

> I don't think this should be translated though

You and I simply don't know (pinging @WaseemAlkurdi since I'm curious about Arabic), but even for a language like Dutch I might opt to localize it as `()` instead of `[]`.

Reply:
I copied that from the toc code, which is "translated", but the footer code is not (e.g. you cannot translate `"%d / %d"` to `"%d : %d"`)

Reply:
That would be a bug then.

Reply:
@Frenzie We would flip it normally for RTL, like a page number:
```
[2 / 14] 3
```
(page 2 out of 14 in non-linear flow number 3)
would become:
```
3 [14 / 2]
```

Reply:
That's the self-evident part. But you'd use those symbols? :-)

Reply:
@Frenzie Ah. The circular brackets would be better, as you stated. Square brackets have no place in user-facing stuff (imagine them being used in a literary text, a novel of some sort?)